### PR TITLE
package: enable cloud-hypervisor for arm64

### PR DESCRIPTION
**Pull request by jongwu**
Now, cloud-hypervisor is capable to work on arm64. it's time to
enable it in kata for arm64.
as cloud-hypervisor can only use virtio-fs, a new patch should be
applied to kernel for virtiofs and some config should be removed
temporarily.

Fixes: #446
Signed-off-by: Jianyong Wu <jianyong.wu@arm.com>

@jodh-intel  @devimc @Pennyzct

### CODE REVIEW

File: tools/packaging/kernel/build-kernel.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ readonly script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 kata_version="${kata_version:-}"
 
 #project_name
-readonly project_name="kata-containers"
+readonly project_name="kata-containers/kata-containers/tools"
```
Comment (devimc) [inline tools/packaging/kernel/build-kernel.sh:20]: please don't modify this variable or the install path will change

https://github.com/kata-containers/kata-containers/blob/85584427e08d3615e35330dc51f9db1cf2b8079c/tools/packaging/kernel/build-kernel.sh#L405

File: tools/packaging/kernel/configs/fragments/arm64/ptp.conf [hunk 2]
```diff
@@ -3,5 +3,5 @@
 # The implementation of ptp_kvm on arm is one experimental feature,
 # you need to apply private patches to enable it on your host machine.
 # See https://github.com/kata-containers/packaging/pull/998 for detailed info.
-CONFIG_PTP_1588_CLOCK=y
-CONFIG_PTP_1588_CLOCK_KVM=y
+#CONFIG_PTP_1588_CLOCK=y
```
Comment (devimc) [inline tools/packaging/kernel/configs/fragments/arm64/ptp.conf:6]: please document why you are commenting out this

Comment (jongwu) [reply tools/packaging/kernel/configs/fragments/arm64/ptp.conf:6]: ok

File: tools/packaging/kernel/build-kernel.sh (hunk 1 above)
Comment (jongwu) [reply tools/packaging/kernel/build-kernel.sh]: ok

**Comment by jongwu** [comment]
@devimc thanks, help trigger ubuntu ci.

**Comment by jongwu** [comment]
thanks @bergwolf !

**Comment by jongwu** [comment]
it seems the ci failure is no related with this PR @bergwolf @devimc

**Pull request merged**